src/templates/feedback.html

```diff
@@ -17,8 +17,8 @@
 {% endmacro %}
 
 <div class="main feedback">
-  <header class="secondary-header">
-    <h2 class="linefit"><span class="linefitted">Feedback</span></h2>
+  <header class="subheader">
+    <h1 class="linefit"><span class="linefitted">Feedback</span></h1>
```

Comment:
Can you change `Feedback` to `{{ _('Feedback') }}`, please. This should use the localised string.